src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -197,7 +205,9 @@ const ActionsPageEvent = ({
       tokensChanged: !!tokenAddresses?.length,
     };
   }, [colonyMetadataHistory, actionData, metadataJSON, eventName, colony]);
-  const roleNameMessage = { id: `role.${values?.roles[eventIndex].id}` };
+  const roleNameMessage = {
+    id: `role.${values?.roles ? values?.roles[eventIndex].id : 'unknown'}`,
```

Comment:
I would make sure that roles[eventIndex] exist

Reply:
This is not my code, I was just trying to fix it since it kept crashing the page when HMR tries to reload it.

But yeah, I'll make the check.